app/redux/reducers/AnalysisReducer.js

```diff
@@ -12,25 +12,30 @@ import {
     SAVE_EXCLUDES_TAGS,
     DISMISS_INCLUDES_TAGS,
     DISMISS_EXCLUDES_TAGS,
+    CALC_ONE_RM,
 } from 'app/ActionTypes';
 
 const defaultState = {
     isEditing1RMExercise: false,
     isEditingIncludeTags: false,
     isEditingExcludeTags: false,
-    e1RMVelocity: .01,
+    velocitySlider: .01,
```

Comment:
can you explain the switch from e1RMVelocity to velocitySlider given that you're still keeping e1RMVelocity?

Reply:
The Velocity slider and the e1rm velocity are separate now, as we are no longer dynamically generating e1rm, but rather on pressing the calculate button. We do not want the e1rm velocity to change when the user slides velocity only when they calculate.